robottelo/vm.py

```diff
@@ -249,6 +260,9 @@ def install_katello_agent(self):
             raise VirtualMachineError('Failed to install katello-agent')
         gofer_check = self.run('service goferd status')
         if gofer_check.return_code != 0:
+            self.run('service goferd start')
```

Comment:
I hope there's no need anymore - bz is finally on_qa, mentioned updated packages are already available downstream. Will try to revert this change and see how it goes.